File: runs/A/bmad/02/code/seed-a-frontend/src/app/core/api/event-detail.ts
export interface PriceCategory {
  id: number;
  name: string;
  preis: number;
}

export interface EventDetail {
  id: number;
  titel: string;
  beschreibung: string;
  dauerMinuten: number;
  altersfreigabe: number;
  spielstaette: string;
  raum: string;
  zeitpunkt: string;
  preiskategorien: PriceCategory[];
}

File: runs/A/bmad/02/code/seed-a-backend.Api/Application/EventDetail.cs
namespace seed_a_backend.Api.Application;

/// <summary>Projektion für die Veranstaltungsdetails, DTO-Mapping auf die API-Form bleibt im Controller (AD-5).</summary>
public record EventDetail(
    int Id,
    string Titel,
    string Beschreibung,
    int DauerMinuten,
    int Altersfreigabe,
    string Spielstaette,
    string Raum,
    DateTime Zeitpunkt,
    IReadOnlyList<PriceCategorySummary> Preiskategorien);
